modules/permutiveRtdProvider.md

```diff
@@ -42,12 +42,15 @@ Key-values details for custom parameters:
 
 * **Audience Connector:** You'll need to define which bidders should receive Audience Connector cohorts. You need to include the `ID` of any bidder in the `acBidders` array. Audience Connector cohorts will be sent in the `p_standard` key-value.
 
-
 ## Parameters
-| Name              | Type                 | Description        | Default        |
-| ----------------- | -------------------- | ------------------ | ------------------ |
-| name              | String               | This should always be `permutive` | - |
-| waitForIt         | Boolean              | Should be `true` if there's an `auctionDelay` defined (optional) | `false` |
-| params            | Object               |                 | - |
-| params.acBidders  | String[]             | An array of bidders which should receive AC cohorts. | `[]` |
-| params.maxSegs    | Integer              | Maximum number of cohorts to be included in either the `permutive` or `p_standard` key-value. | `500` |
+| Name                    | Type                 | Description        | Default        |
+| ----------------------- | -------------------- | ------------------ | ------------------ |
+| name                    | String               | This should always be `permutive` | - |
+| waitForIt               | Boolean              | Should be `true` if there's an `auctionDelay` defined (optional) | `false` |
+| params                  | Object               |                 | - |
+| params.acBidders        | String[]             | An array of bidders which should receive AC cohorts. | `[]` |
+| params.maxSegs          | Integer              | Maximum number of cohorts to be included in either the `permutive` or `p_standard` key-value. | `500` |
+| params.transformations  | Object[]             | An array of configurations for ORTB2 user data transformations 
```

Comment:
Are the transformations intended to map standard cohorts to different IAB taxonomies, or something more generic?

Reply:
For the parameter I've modelled it very generally, as any transformation on the `user.data` object without saying anything specific about what the transformation would do. WDYT?

Reply:
I think it's essential to document the different transformations available, so that people can effectively use the module.

My preference would be to have something less general:
```
{
  "iabTaxonomies": {
    "<taxonomyId>": {
      "<standardCohort>": "<iabCohort>"
    }
  }
}
```

I _think_ this would simplify the implementation, and support other IAB taxonomies out-of-the-box. It would also be easier to write documentation for.

That being said, main emphasis of this comment is +1 for documenting each transformation, rather than the configuration specifics (which is more just another approach for you to weigh up).

Reply:
I was thinking about it differently: I don't think it's guaranteed we're going to stick with this approach beyond the Hearst test (e.g. if we want to run this properly for all standard cohorts, I doubt we'll ask publishers to include the full list of cohorts in their config) and I'm expecting us to provide some specific documentation to Hearst (separately to this readme) that includes the config code they need to include as well as mentioning that this will apply to all bidders. 

I totally agree that we should document the transformations we support but wonder if it's appropriate to add something that might only be accurate for the short duration of the Hearst test?

Reply:
Ok, but I'm not sure Prebid will see it this way. My guess is that they will want functionality merged into the SDK to be general and not tied to a specific use case. But happy to raise as is and not speculate.

That being said, if we don't want to document the IAB taxonomy transformation, what is the point of documenting transformations?

If we did want to the document the IAB taxonomy transformation, we could mark it as alpha in the docs and subject to change.

Reply:
I agree it's not that useful to have an entry on transformations when we don't say much about how to use it. But let's see what Prebid say.